sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -10,9 +10,9 @@
     <projectUrl>http://vs.sonarlint.org</projectUrl>
     <icon>images\sonarlint_64.png</icon>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <summary>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio and/or the SonarQube platform.</summary>
-    <description>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio (http://vs.sonarlint.org) and/or the SonarQube platform (http://www.sonarqube.org).</description>
-    <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.13.0.0</releaseNotes>
+    <summary>Analyzers which spot bugs, vulnerabilities and code smells in your code. To have a better integration with SonarQube and/or SonarCloud, you can use the SonarLint for Visual Studio extension.</summary>
+    <description>Analyzers which spot bugs, vulnerabilities and code smells in your code. To have a better integration with SonarQube (https://www.sonarqube.org/) and/or SonarCloud (sonarcloud.io/), you can use the SonarLint for Visual Studio extension (https://www.sonarlint.org/visualstudio/).</description>
```

Comment:
```suggestion
    <description>Analyzers for C# and VB.NET languages which spot bugs, vulnerabilities and code smells in your code. You can use the SonarLint for Visual Studio extension (https://www.sonarlint.org/visualstudio/) to detect quality issues in your IDE and have a better integration with SonarQube (https://www.sonarqube.org/) and/or SonarCloud (https://sonarcloud.io/).</description>
```

Reply:
I just realized that we cannot say "analyzers for C# and VB.NET" because each language comes with it's own nuget